heroku/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\btros\\Anaconda3\\python.exe"
```

Comment:
I'm thinking it would be best to not have vscode configurations in version control, seeing as there will be significant differences between our systems.

Going to request changes to the .gitignore to include the .vscode directory. Would also be good to do the same for the master .gitignore. We can discuss which one of us can do that. Probably best to do that after this is merged into dev.